### ISSUE TITLE
Need Suggestions!

### ISSUE THREAD

**Issue by Simpleboy353** [Simpleboy353/REAPER-2.0#15]
Hello to everyone reading this, first of all thank you for all the love you have shown to this repo! Secondly, I have opened this issue, so that you can suggest features that could be added to this bot! For now, I am thinking of adding music commands to the repo. If you want music, please comment down here and let me know or you can suggest any feature by commenting the feature name an a description for it!

**Comment by HACKERPROTM** [comment]
add a music command like loop or volume but not in diffrent files in one

**Comment by HACKERPROTM** [comment]
its easy you just use aliases

**Comment by Simpleboy353** [comment]
> add a music command like loop or volume but not in diffrent files in one

Are you referring to a complete music system or just one of these mentioned commands?

**Comment by HACKERPROTM** [comment]
> > add a music command like loop or volume but not in diffrent files in one
> 
> Are you referring to a complete music system or just one of these mentioned commands?

Look at my open repo and check what i mean

**Comment by Simpleboy353** [comment]
> > > add a music command like loop or volume but not in diffrent files in one
> > 
> > 
> > Are you referring to a complete music system or just one of these mentioned commands?
> 
> Look at my open repo and check what i mean

ok I got what you are saying but I don't think I would do something like that because I have ready music code from my music bot which I don't use anymore, so I was thinking about adding that to this repo.

**Comment by nightmxrethedev** [comment]
music and logging features?

**Comment by Simpleboy353** [comment]
> music and logging features?

I have changed my plans for music, as i need to update the music code to v13. Talking about logging, I'll try to add that in the next update. Also the update could be delayed for now, as currently my exams are going on.

**Comment by nightmxrethedev** [comment]
oh, nice. all the best for ur exams!

**Comment by Toxic-Razor** [comment]
Are you planning on adding music commands with Lavalink ?

**Comment by Simpleboy353** [comment]
> Are you planning on adding music commands with Lavalink ?
> 
> I would recommend using lavalink as the quality of the music will be high and its lag free, most of the large bots use it.

Sorry for the late reply, as i said earlier I have changed my plans regarding music. Music won't be added till v7.0. I am still planning about what to use for music. I would go either for Discord-Player or Lavalink

**Comment by nightmxrethedev** [comment]
Suggestions for next update(s):
- Integrated ModMail System
- Music
- Web Dashboard 
- Reaction Roles

These were just some cool suggestions about what could be added to the bot to make it even better than it already is.

**Comment by Simpleboy353** [comment]
Thank for the suggestions! I will try to add the stuff which I can!

**Comment by nightmxrethedev** [comment]
#RoadToMakeInfintyTheBestDiscordBot xD

**Comment by Simpleboy353** [comment]
btw expect an update today or tomorrow!

**Comment by nightmxrethedev** [comment]
*uwu*

**Comment by Simpleboy353** [comment]
Update Released

**Comment by AhmadShkour71** [comment]
Just, Thx You Bro

**Comment by Simpleboy353** [comment]
> Just, Thx You Bro

ok.. but thanks for what?

**Comment by nightmxrethedev** [comment]
maybe for releasing the new update?

who knows xD

**Comment by Simpleboy353** [comment]
xD

**Comment by AhmadShkour71** [comment]
> > Just, Thx You Bro
> 
> ok.. but thanks for what?

Nothing 
Just thx thank  you 😊

**Comment by yawn999** [comment]
warn commands?

**Comment by mrmaton** [comment]
a verification system with discord captcha and role after

**Comment by DanielJavaScript** [comment]
add ticket system

**Comment by DanielJavaScript** [comment]
Web dashboard:)))

**Comment by 1stminhcar** [comment]
> 
> 
> Web dashboard:)))

hey

**Comment by DanielJavaScript** [comment]
Suggestions for next update(s):

 * ModMail System
 * Web Dashboard
 * Ticket System

**Comment by Blizzard2875** [comment]
help command e.g !help ping sends an embed with the ping commands description, usage, alias etc

**Comment by nightmxrethedev** [comment]
> help command e.g !help ping sends an embed with the ping commands description, usage, alias etc

that already exists [here](https://github.com/Simpleboy353/REAPER-2.0/blob/master/Commands/Info/help.js)

**Comment by Simpleboy353** [comment]
> add ticket system

Separate repo available for this: https://github.com/Simpleboy353/Tickets-Bot

**Comment by Simpleboy353** [comment]
> Suggestions for next update(s):
> 
> * ModMail System
> * Web Dashboard
> * Ticket System

ModMail: Not sure about this
Web Dashboard: No plans
Tickets: Seperate Repo available that uses same code structure, just merge that code with repear's code

**Comment by Simpleboy353** [comment]
> warn commands?

Will be implemented soon

**Comment by Simpleboy353** [comment]
> help command e.g !help ping sends an embed with the ping commands description, usage, alias etc

Already implemented in v7.1.0

**Comment by Simpleboy353** [comment]
> a verification system with discord captcha and role after

Will be implemented soon

**Comment by Simpleboy353** [comment]
> Invite Tacker where it collect's the user and how many invite's they have made/people have used.

No plans

**Comment by Simpleboy353** [comment]
> Hi. You can use [Giveaways](https://www.npmjs.com/package/discord-giveaways) package and for bot code you can refer [This](https://github.com/ZeroDiscord/Giveaway)

Not sure but may I'll add it

**Comment by 1stminhcar** [comment]
> 
> 
> > Hi. You can use [Giveaways](https://www.npmjs.com/package/discord-giveaways) package and for bot code you can refer [This](https://github.com/ZeroDiscord/Giveaway)
> 
> Not sure but may I'll add it

it work out quite well

**Comment by Natesplanet** [comment]
Add a feature where ONLY SOME ROLES CAN USE SOME COMMANDS. It would be rly helpful :)

But other then that thanks SimpleBoy for making this for the community

**Comment by FriendlyCodehelper** [comment]
Invite tracker system

**Comment by Natesplanet** [comment]
> Invite tracker system

@Simpleboy353  Already added a state ment to a tracked system
![image](https://user-images.githubusercontent.com/92893096/141283771-217f132f-39b7-41be-ab26-c05b474cc543.png)

**Comment by FriendlyCodehelper** [comment]
Okay, is there any antighost ping system i recommend it if not there

**Comment by Natesplanet** [comment]
@Simpleboy353  You should add a ticket system!!

**Comment by Simpleboy353** [comment]
> @Simpleboy353 You should add a ticket system!!

refer to https://github.com/Simpleboy353/Tickets-Bot, this bot uses the same code structure, you can easily merge it with

**Comment by Natesplanet** [comment]
Thanks Simpleboy!

**Comment by Natesplanet** [comment]
Also, one more thing @Simpleboy353  can you check if some of the codes in the commands folder here https://github.com/AgentRocky/PIper can be merged into Reaper?

Thanks :D

**Comment by Simpleboy353** [comment]
Yes you can, but you will have to remove the `module.exports.help` part also change the `module.exports.run` to ```js
module.exports = {
    name: "commandname",
    description: "description",
    run: async(client, message, args) =>{
    // Code here
    }
}
```

**Comment by Natesplanet** [comment]
@Simpleboy353  How would you merge both 3 Repositories together

[https://github.com/Simpleboy353/Tickets-Bot](url)
[https://github.com/AgentRocky/PIper](url)
And My Repaer
[https://github.com/natesplanet/N-Bot](url)

**Comment by Natesplanet** [comment]
also, @Simpleboy353  do you know how to code an invite tracker into Reaper?

**Comment by CodeXSane** [comment]
Ticket system.
Reaction roles.
Web dashboard.
Prefix music commands.
